compiler/damlc/tests/daml-test-files/InterfaceNotSupported.daml

```diff
@@ -1,4 +1,4 @@
--- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
we should either drop the no autocopyright from this directory (not sure why it’s there in the first place) or drop all the copyright headers. Not in this PR though.

Reply:
FWIW removing the `NO_AUTO_COPYRIGHT` file results in the copyright header script crashing on an invalid unicode file. I believe it is intentional that the file is invalid (to test how the compiler reacts on that), and I guess it was easier to ignore the entire folder than fix the script (hence my "too coarse-grained" comment).